samples/bookinfo/src/ratings/ratings.js

```diff
@@ -118,7 +148,18 @@ dispatcher.onGet('/health', function (req, res) {
   res.end(JSON.stringify({status: 'Ratings is healthy'}))
 })
 
+function putLocalReviews (productId, ratings) {
+  userAddedRatings[productId] = {
+    id: productId,
+    ratings: ratings
+  }
+  return getLocalReviews(productId)
+}
+
 function getLocalReviews (productId) {
+  if(typeof userAddedRatings[productId] !== 'undefined') {
```

Comment:
nit: `if(` -> `if (`

Reply:
fixed in bb38dabf8956b3702ecb8922f0c0cf6e3d1a652d